packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -79,6 +102,16 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     fileDownloadHandler
   } = props;
 
+  const defaultTimeStamp = message.createdOn
+    ? generateDefaultTimestamp(message.createdOn, showDate, strings)
+    : undefined;
+
```

Comment:
nitty, nitty, nit: I'd drop the newlines between these variables (they're all related)